### PR TITLE
chore: chore: Improve Cockpit deployment configurability

### PR DESCRIPTION
**Pull request by brasseld**
Closes gravitee-io/gravitee-cockpit#763

### CODE REVIEW

File: am/values.yaml [hunk 1]
```diff
@@ -173,6 +173,15 @@ api:
   # https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
   priorityClassName: ""
 
+  env: []
```
Comment (jgiovaresco) [inline am/values.yaml:176]: Did you see that is an AM file, should we include it in the PR? 🤔

File: cockpit/values.yaml [hunk 2]
```diff
@@ -127,14 +127,60 @@ api:
...
+    tcpSocket:
+      port: http
+    initialDelaySeconds: 30
+    periodSeconds: 30
+    failureThreshold: 3
+
+  podAnnotations: {}
+    # iam.amazonaws.com/role: es-cluster
+
+  # How long to wait for APIM Management API pods to stop gracefully
```
Comment (jgiovaresco) [inline cockpit/values.yaml:168]:
```suggestion
  # How long to wait for Cockpit Management API pods to stop gracefully
```

**Review by jgiovaresco — APPROVED** [review]

File: am/values.yaml (hunk 1 above)
Comment (brasseld) [reply am/values.yaml:176]: Yep, sorry for that. I've missed this from my previous AM PR :(

**Comment by brasseld** [comment]
FP

**Pull request merged**